src/lwc/geGiftEntryFormApp/geGiftEntryFormApp.js

```diff
@@ -82,6 +74,13 @@ export default class GeGiftEntryFormApp extends NavigationMixin(LightningElement
             this.sObjectName === DATA_IMPORT_BATCH_OBJECT.objectApiName;
     }
 
+    connectedCallback() {
+        getPaymentTransactionStatus()
```

Comment:
Can we rename this to "getPaymentTransactionStatusEnum" or "getPaymentTransactionStatusValues"? This makes it seem like you are getting the value of of the transaction status, which is misleading